features/polyExplorer/src/components/clusterStories/mauLineChart.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from "react";
+import { ChipGroup, LineLegend, PolyChart } from "@polypoly-eu/poly-look";
+import i18n from "../../i18n";
+
+import "./mauLineChart.css";
+
+const facebookColor = "#3749A9";
+const otherColor = "#3BA6FF";
+
+const otherColorSet = { color: otherColor, gradient: "other-gradient" };
+const facebookColorSet = {
+    color: facebookColor,
+    gradient: "facebook-gradient",
+};
+
+const gradients = [
+    {
+        id: "facebook-gradient",
+        type: "linearGradient",
+        stops: [
+            { offset: "0%", color: facebookColor, opacity: 0.1 },
+            { offset: "100%", color: facebookColor, opacity: 0.5 },
+        ],
+    },
+    {
+        id: "other-gradient",
+        type: "linearGradient",
+        stops: [
+            { offset: "0%", color: otherColor, opacity: 0.1 },
+            { offset: "100%", color: otherColor, opacity: 0.5 },
+        ],
+    },
+];
+
+const coloring = {
+    Signal: otherColorSet,
```

Comment:
This makes me a wee bit nervous - where are those strings ("Signal", "Facebook Messenger" and so on) coming from? Looks like displayed strings to me, because in the data, it seems like each messenger has a ppid. I'm worried that we'd have to make changes in _two_ places if one of those (display) names here changes. But I'd be fine with it for now.

Reply:
Strings are gone 👍